app/src/main/java/org/openhdp/hdt/ui/tracking/addCounter/AddStopwatchBottomSheetFragment.kt

```diff
@@ -0,0 +1,129 @@
+package org.openhdp.hdt.ui.tracking.addCounter
+
+import android.app.Activity
+import android.content.res.ColorStateList
+import android.graphics.Color
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.view.inputmethod.EditorInfo
+import android.view.inputmethod.InputMethodManager
+import android.widget.PopupMenu
+import android.widget.TextView
+import androidx.core.content.ContextCompat
+import androidx.core.widget.addTextChangedListener
+import androidx.fragment.app.viewModels
+import dagger.hilt.android.AndroidEntryPoint
+import org.openhdp.hdt.R
+import org.openhdp.hdt.databinding.FragmentAddElementBinding
+import org.openhdp.hdt.ui.base.RoundedSheetDialogFragment
+import org.openhdp.hdt.ui.tracking.TrackingItem
+
+
+@AndroidEntryPoint
+class AddElementBottomSheetFragment : RoundedSheetDialogFragment() {
+
+    interface Listener {
+        fun onAdded(item: AddStopwatchViewState)
+    }
+
+    private var popupMenu: PopupMenu? = null
+    var listener: Listener? = null
+
+    private val viewModel: AddStopwatchViewModel by viewModels()
+
+    private lateinit var binding: FragmentAddElementBinding
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        binding = FragmentAddElementBinding.inflate(inflater, container, false)
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
```

Comment:
I think that this function is the same as in `Categories`. Maybe we could create utility function that will be call in this override fun ?